calico/maintenance/monitor/monitor-component-metrics.md

```diff
@@ -31,7 +31,7 @@ You can configure Felix, Typha, and/or kube-controllers to provide metrics to Pr
 
 ### Before you begin...
 
-In this tutorial we assume that you have completed all other introductory tutorials and possess a running Kubernetes cluster with {{site.prodname}}, calicoctl and kubectl installed.
+In this tutorial we assume that you have completed all other introductory tutorials and possess a running Kubernetes cluster with {{site.prodname}}, calicoctl or apiserver and kubectl installed.
```

Comment:
I think this can just be "running a Kubernetes cluster with {{site.prodname}}"

Reply:
Can we have a tab up here as well which explains preqreuisites?

Something like this:

```
You can either use kubectl or calicoctl to perform the following steps. Depending on which tool you would like to use, make sure you have the necessary prerequisites as shown below.

<tabs
kubectl
>
< tab 
calicoctl
>
```